demos/kqueen_azure-demo.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+################################################
+# include the demo-magic script
+################################################
+# demo-magic.sh is a handy shell script that enables you to script repeatable demos in a bash environment so you don't have to type as you present
```

Comment:
I wouldn't understand this description if I haven't seen how it works. I think smth like that is more clear:
`..in a bash environment. It simulates typing of your commands, so you don't have to type them by yourself when you are presenting`